konfera/event/views.py

```diff
@@ -1,21 +1,35 @@
+import logging
+
 from datetime import timedelta
+from smtplib import SMTPException
 
+from django import VERSION
+from django.conf import settings as django_settings
```

Comment:
we dont need this import, use settings from konfera, you already have that import down below...